### ISSUE TITLE
XPU operator performance enhancement

### ISSUE THREAD

**Issue by xytintel** [intel/torch-xpu-ops#1133]
#### Performance enhancement [not exposed]
- [batch_normalization: Introduce vectorization optimization in the batch norm elementwise kernel](https://github.com/intel/torch-xpu-ops/pull/933)
- [upsample_biliear2d channel last backward optimization](https://github.com/intel/torch-xpu-ops/pull/950)
- [group norm forward vectorization optimization](https://github.com/intel/torch-xpu-ops/pull/1116)
- [max pool operator level optimization](https://github.com/intel/torch-xpu-ops/pull/1127)
